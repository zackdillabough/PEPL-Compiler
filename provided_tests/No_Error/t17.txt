POLY D = x + 2 x + 9 x + 5 x^2 + 10;
POLY B(a, b, c, d) = a^0 + b^1 + c^2 + d^3;
POLY A(e) = e + 5;
START
INPUT P;
INPUT Q;
INPUT R;

B(P, Q, Q, R);

D(1);
A(Q);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P))))))))))
))))))))));


INPUT P;
INPUT Q;
INPUT R;

B(P, Q, Q, R);

D(1);
A(Q);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P))))))))))
))))))))));

INPUT P;
INPUT Q;
INPUT R;


B(P, Q, Q, R);

D(1);
A(Q);

D(R);

A(A(
A(A(
A(A(
A(A(A(A(A(A(
A(A(





A(A(


A(A(

A(A(P))))))))))
))))))))));


11 22 13 14 9 6 5 2 1 2 6 6 7 3 1 3 0 1 2 1
