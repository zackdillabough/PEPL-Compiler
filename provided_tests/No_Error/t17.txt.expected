2704
27
27
1011
111
307
27
14
262
114
8
27
7
27
105
